openrouteservice/src/main/java/org/heigit/ors/api/responses/centrality/json/JSONCentralityLocation.java

```diff
@@ -0,0 +1,41 @@
+package org.heigit.ors.api.responses.centrality.json;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.vividsolutions.jts.geom.Coordinate;
+import io.swagger.annotations.ApiModelProperty;
+import org.heigit.ors.util.FormatUtility;
+
+import java.util.Map;
+
+public class JSONCentralityLocation {
```

Comment:
Prefer using proper CamelCase for readability: JsonCentralityLocation. Otherwise, the word boundary between JSON and Centrality is blurred. The same should be applied to many names in this PR and will not be commented seperately in this review.

Reply:
I agree with the word boundary blurring causing the class name to be hard to read.

`JSONCentralityLocation` was chosen to be consistent with all other response-classes, i.e. `GPXLink`, `JSONSegment` or `JSONBasedIndividualMatrixResponse`.
Any changes here wouldn't affect the rest of the code base, but would be inconsistent. Will this inconsistency be addressed by refactoring all other class names at some point?

Reply:
At least it should be addressed in the future.